sda/cmd/finalize/finalize.go

```diff
@@ -144,24 +143,24 @@ func main() {
 			completeMsg, _ := json.Marshal(&c)
 			err = schema.ValidateJSON(fmt.Sprintf("%s/ingestion-completion.json", conf.Broker.SchemasPath), completeMsg)
 			if err != nil {
-				log.Errorf("Validation of outgoing message failed, reason: (%v)", err)
+				log.Errorf("Validation of outgoing message ingestion-completion failed, reason: (%v). Message body: %s\n", err, string(completeMsg))
 
 				continue
 			}
 
 			accessionIDExists, err := db.CheckAccessionIDExists(message.AccessionID, fileID)
 			if err != nil {
-				log.Errorf("CheckAccessionIdExists failed, reason: %v ", err)
+				log.Errorf("CheckAccessionIdExists failed, file-id: %s, reason: %v ", fileID, err)
 				if err := delivered.Nack(false, true); err != nil {
-					log.Errorf("failed to Nack message, reason: (%v)", err)
+					log.Errorf("failed to Nack message, reason: %v", err)
 				}
 
 				continue
 			}
 
 			switch accessionIDExists {
 			case "duplicate":
-				log.Debugf("Seems accession ID already exists (corr-id: %s, accessionid: %s", delivered.CorrelationId, message.AccessionID)
+				log.Errorf("accession ID already exists in the system. (accession-id: %s)\n", message.AccessionID)
```

Comment:
I think this is a case where we would like to have the corrID as well, for quickly identifying the failed file. Otherwise we would need to dig into the error queue (which if it contains more than a few tenths of  messages becomes useless).

Reply:
I agree, I will add the fileID in this log message.

Reply:
fixed in the commit: 44baa904